modules/components/pages/inputs/postgres_cdc.adoc

```diff
@@ -158,6 +158,44 @@ For full visibility of all data updates, you can also choose to stream https://w
 ALTER TABLE large_data REPLICA IDENTITY FULL;
 ```
 
+== Data mappings
+
+The following table shows how selected PostgreSQL data types are mapped to data types supported in Redpanda Connect. All other data types are mapped to string values.
+
+|===
+| PostgreSQL data type | Bloblang value
+
+| TEXT, TIMESTAMP, UUID, VARCHAR
+| JSON strings, for example: `this data`
+
+| BOOL
+| Boolean JSON fields, for example: `true` or `false`
+
+| Numeric types (INT4)
+| JSON number types, for example: `1`.
+
+| JSONB
+| JSON objects, for example: `{{{ "message": "message text" }}}`
```

Comment:
Are the three curly braces intended?

Reply:
Good spot. Updated, and I've checked the formatting of the other data types.